build.gradle.kts

```diff
@@ -204,9 +204,8 @@ tasks {
 	}
 
 	test {
-		
-    	    	configure<JacocoTaskExtension> {
-       		isEnabled = !experimentalBuild
```

Comment:
This is not visible in the GitHub web interface, but indentation here is using both spaces and tabs.